src/rez/cli/test.py

```diff
@@ -67,3 +95,75 @@ def command(opts, parser, extra_arg_groups=None):
 
         if returncode:
             sys.exit(returncode)
+
+
+def is_dev_run(name):
+    """
+    Decides if is a dev_run, based on the package name
+    Args:
+        name: name of package
+
+    Returns: True or False
+    """
+    return not name or name == '.'
```

Comment:
Too fragile. As mentioned earlier, I think it'd be better if developer mode is enabled only if PKG is not provided. Right now is odd because PKG can be a package string, or ".", but not any other valid path. There is too much variability in what PKG can be.

Reply:
agree